### PR TITLE
Add New Weapon Values for Progression

### PR DESCRIPTION
**Pull request by Chilly021**
Give other weapon paths a shot.

### CODE REVIEW

**Comment by Chilly021** [comment]
This is for discussion..  I am thinking about base damage not always being 1 for progression weapons.

File: Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs [hunk 1]
```diff
@@ -17,10 +17,10 @@ public partial class WoodenFlail : Flail
 		public override int MinimumDamage => 1;
 
 		/// <inheritdoc />
-		public override int MaximumDamage => 6;
+		public override int MaximumDamage => 1;
 
 		/// <inheritdoc />
-		public override int BaseArmorBonus => 1;
+		public override int BaseArmorBonus => 10;
```
Comment (sproatba) [inline Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs:23]: I think you got the wrong property for the 10 on this one

**Review by sproatba — CHANGES_REQUESTED** [review]

File: Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs (hunk 1 above)
Comment (Chilly021) [reply Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs:23]: Oof yep.

Comment (Chilly021) [reply Source/Kesmai.Server/Game/Items/Equipment/Weapons/Flails/WoodenFlail.cs]: corrected.

**Review by sproatba — APPROVED** [review]

**Review by zrikzlok — APPROVED** [review]

**Review by Leolus — APPROVED** [review]

**Pull request merged**